nequip/model/_eng.py

```diff
@@ -53,7 +53,6 @@ def EnergyModel(config) -> SequentialGraphNetwork:
     layers["total_energy_sum"] = (
         AtomwiseReduce,
         dict(
-            reduce="sum",
```

Comment:
maybe it is good to keep the `reduce` argument since the reduce mode here shouldn't depend on the default value.